### DISCUSSION TITLE
How can I avoid automatic yaw control?

### DISCUSSION THREAD

**Discussion by akdrn16** [NatLabRockies/floris#620] [Q&A] [answered]
Hello, I am a graduate student studying wind farm layout using FLORIS. 
Thank you for providing such a great open source tool.

I found out that when setting yaw_angle to None or not providing any input during wake calculation, yaw rotates automatically in the inflow direction. 
But I would like to calculate the power output of the wind farm without any yaw control. 
Could you please advise me on how to set the yaw_angle to achieve this? 

I tried setting the yaw_angle to the opposite value of the inflow wind direction to eliminate the yaw control effect, but I found that the get_farm_power value was NaN in many time steps during the calculation of the time series output data.
How can I keep the yaw fixed to North (0 degrees) without rotating in the inflow direction depending on the wind direction changes?

Also, can we consider the yaw rotation in the inflow direction as greedy control?

Thanks in advance :)

**Comment by paulf81** [comment, marked as answer]
Wind turbines will normally orient themselves into the incoming wind direction when operating.  Turbines can operate with some amount of misalignment to the incoming direction, but at a certain limit most turbines will shut themselves off.  At some amount of misalignment it becomes difficult to model (perhaps after 40 degrees), and that is why the result starts to include NaN values.

**Reply by akdrn16** [reply]
Thank you for your quick response!
I have an additional question. So, is it correct to say that the Baseline AEP is a value that is calculated as the yaw angle is aligned according to the wind direction? I thought the meaning of the Baseline was the AEP output with a fixed yaw angle, thinking that it meant the prevailing wind direction derived from the windrose in the wind farm layout. However, is it correct to interpret the AEP output with Greedy control performed as the Baseline AEP?

**Reply by paulf81** [reply]
I think that's right.  Baseline can me a lot of things I think depending on what is being studied, but a common use case for FLORIS is assessing the value of wake steering, in which the turbines are sometimes intentionally offset from the incoming direction to benefit waked turbines downstream (farm control) versus every turbine aligned, which is in a sense a greedy approach.

**Reply by akdrn16** [reply]
Thank you for your kindness.🤓
Your explanation has cleared up many confusing concepts for me!  
I really appreciate your help and thank you again.🤩

**Reply by paulf81** [reply]
happy to help and good luck!
